src/bin/post-comment-to-pr.py

```diff
@@ -30,9 +27,12 @@
     # Construct comment body
     now = datetime.now().strftime('%Y.%m.%d %H:%M:%S')
     title = '### Test instance is ready :rocket:'
-    msg = (':new_moon: [{0}]({1}{0})\n\n'
-           ':watch: {2}').format(
-            test_instance, TEST_INSTANCE_PREFIX, now)
+    msg = (':new_moon: [{0}](https://www-dev.greenpeace.org/test-{0}) | '
+           '[admin](https://www-dev.greenpeace.org/test-{0}/wp-admin/) | '
+           '[CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-{0}) | '
+           '[composer-local.json](https://github.com/greenpeace/planet4-test-{0}/blob/main/composer-local.json)\n\n'
+           ':watch: {1}').format(
+            test_instance, now)
     body = '{0}\n\n{1}'.format(title, msg)
 
```

Comment:
It also means that if you create a PR, and push a new commit before the label is added, it will be deployed to another instance, potentially multiple times.

Reply:
I created a PR to address this issue https://github.com/greenpeace/planet4-circleci/pull/55